src/endpoints/createLanguagesEndpoint.js

```diff
@@ -18,11 +19,17 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<LanguageModel>> =>
       }
       return `${baseUrl}/${params.city}/de/wp-json/extensions/v3/languages`
     })
-    .withMapper((json: Array<JsonLanguageType>) => json
-      .map(language => new LanguageModel(
-        language.code,
-        language.native_name
-      ))
-      .sort((lang1, lang2) => lang1.code.localeCompare(lang2.code))
-    )
+    .withMapper((json: Array<JsonLanguageType>) => {
+      return json
+        .map(language => {
+          if (!language.native_name) {
```

Comment:
Please remove this and revert since a separate endpoint was created, instead there should be two separate json types.